chart/templates/crds/chaos.datadoghq.com_disruptions.yaml

```diff
@@ -1,5 +1,7 @@
-
----
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
```

Comment:
Seems that CI is sad when this isn't verbatim the same file generated by `controller-gen`. I think on our end it's fine to not include the copyright header here. Could you take this change out as well? Thanks!

Reply:
I can revert that, sure.
Unfortunately a Helm install doesn't work without that though. [This](https://github.com/DataDog/chaos-controller/blob/91c4fca8c49a289ac5ca6b6412749b7ce0cf73a8/chart/templates/crds/chaos.datadoghq.com_disruptions.yaml#L2) line causes issues.

Reply:
Hmm, let me see what I can do w/ the CI check then

Reply:
@Devatoria What are your thoughts on ^ ?

Reply:
This file is autogenerated. The CI re-generates it and compares it to the committed one to be sure there are no diff. Here, either removing the `---` or adding a header will make a diff. I don't think we have a hand on this automatically added first line `---` anyway. Let me have a look at Helm to see why it would fail on a standard YAML content separator.

Reply:
Could you prove us your `helm` version please? I can't reproduce the error (but I have a lot of other errors related to metadata though).

Reply:
I don't know what to say. I tried many times today and I didn't manage to reproduce either ¯\_(ツ)_/¯
I remember the CRD either not getting deployed or Helm showing an error (not a warning). But as I cannot reproduce anymore I have reverted the change.

Reply:
Alright, thanks for the updates! Feel free to open an issue if the error pops up again at some point.